capitolwords_ng/parsers/crec.py

```diff
@@ -4,13 +4,12 @@
 from datetime import datetime
 
 import boto3
-import re
 from lxml import etree
 
 import spacy
 import textacy
 
-import text_utils
+from parsers.text_utils import *
```

Comment:
`import *` can be risky, especially as people add stuff to that module, should either import each object specifically or prefix those objects with a shortened module name, e.g.:
```python
import parsers.text_utils as text_utils
```